### DISCUSSION TITLE
Modeling sodium ion batteries

### DISCUSSION THREAD

**Discussion by RSPdatascience** [pybamm-team/PyBaMM#4397] [Q&A] [answered]
Hi, I have been looking for software for modeling sodium-ion batteries and I consider pybamm as an option. Am I right to assume that Na-ion batteries' physics is very similar, and models like SPM, SPMe and DFN can be used without substantial modifications, changing only the battery dataset, perhaps modifying an already existing Li-ion battery set? Has anyone attempted to create such a dataset?

**Comment by brosaplanella** [comment, marked as answer]
You are correct. We have not attempted that yet, but we are working on bringing some literature Na-ion parameter values in soon.

**Reply by RSPdatascience** [reply]
Hi, thank you very much for your fast response. Any estimation as to when you may have it? Alternatively, could you provide appropriate references containing such parameters?

**Reply by brosaplanella** [reply]
Hopefully in the next few weeks, with the parameters from this article: https://www.sciencedirect.com/science/article/pii/S0013468621020478

**Reply by RSPdatascience** [reply]
Thank you very much!

**Reply by patelsana27** [reply]
> Hopefully in the next few weeks, with the parameters from this article: https://www.sciencedirect.com/science/article/pii/S0013468621020478

The sodium-ion parameters are now added. However, why do we not have the option of selecting the sodium-ion model directly rather than using lithium-ion base sub-models and then building on it?

**Reply by brosaplanella** [reply]
We just merged the BasicDFN for sodium ion, it's available in the develop branch. It will be included in the next release.

**Reply by RSPdatascience** [reply]
Thank you very much for letting me know it Ferran!

**Reply by DavidMStraub** [reply]
Hi @brosaplanella, I started being interested in Na-ion, too, and discovered BasicDFN. But I'm a bit puzzled - if the structure is exactly the same as Li-ion, why is there a `sodium_ion` module that only contains a stripped-down version? Should everything that is in `lithium_ion` be mirrored there? Or should `lithium_ion` simply become some kind of `alkali_ion`?

**Reply by brosaplanella** [reply]
Hi! That's a very good point. The main reason was precisely to no support all the extra features (mostly degradation) until someone has figured out the proper way to model it.

**Reply by DavidMStraub** [reply]
That makes sense, but thermal models should map 1-to-1 I guess, and SPM/SPMe would be useful, too.
